.eslintrc.js

```diff
@@ -108,6 +108,7 @@ module.exports = {
   plugins: ['jsx-a11y', 'prettier', 'local', 'react-hooks'],
   rules: {
     'prefer-template': 'error',
+    'import/no-unresolved': 'off',
```

Comment:
Eslint started not passing due to the detection of errors like the following
```
68:30 error Unable to resolve path to module './search?raw' import/no-unresolved
```
This may be due to the fact that these files were indexed, but I'm not sure. If you go into the code locally in another branch, IDE will also show the same error. I haven't found a more elegant solution to fix this error, so I turned off the rule.

Reply:
Is there an option to ignore query params? Maybe we could raise this to the ESLint team? Otherwise we could create a follow up issue to dive deeper into this

Reply:
As it turned out, `eslint-import-resolver-webpack` only supports "synchronous" Webpack configs (https://github.com/import-js/eslint-plugin-import/tree/main/resolvers/webpack). How critical is it to use `getPort`? Is it possible to just put port `8030`?

Reply:
> As it turned out, `eslint-import-resolver-webpack` only supports "synchronous" Webpack configs (https://github.com/import-js/eslint-plugin-import/tree/main/resolvers/webpack). How critical is it to use `getPort`? Is it possible to just put port `8030`?

I'm pretty sure the reason for `getPort` is that you can actually have multiple instances of OUI running on different ports - you can replicate by doing `yarn start` in multiple terminal tabs/windows, and the port will auto increment without collisions.